src/runtime/recipe/recipe.ts

```diff
@@ -539,6 +539,7 @@ export class Recipe implements Cloneable<Recipe> {
     };
 
     recipe._name = this.name;
+    recipe.triggers = this.triggers;
```

Comment:
thanks for fixing (and sorry for missing it in the first place)

Reply:
No worries! Had my head scratching for a minute; glad the fix was simple.